### DISCUSSION TITLE
How can I use WebPack to create several HTML files with the same footer and header?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [webpack/webpack#16069] [Q&A]
How can I use WebPack to create several HTML files with the same footer and header? 
I want to output simple old-style static html files but want to use the same html blocks for footer and header.
What is the best way to do it?

Also is there a way just to copy JS files and images?

**Comment by vankop** [comment]
not sure what you mean, you can take a look at https://github.com/jantimon/html-webpack-plugin
